files/entry.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
```

Comment:
During testing, I see

(dcc_check_compiler_masq) /opt/irix/sgug/bin/mips-sgi-irix6.5-gcc is not a symlink

I'm not sure if this is impactful or if it needs to be fixed.